WhaleSpotting/Models/Request/WhaleSpeciesRequest.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace WhaleSpotting.Models.Request;
+
+public class WhaleSpeciesRequest
+{
+    public string ImageUrl { get; set; }
+    public string Name { get; set; }
+    public TailType TailType { get; set; }
+    public TeethType TeethType { get; set; }
+    public WhaleSize Size { get; set; }
```

Comment:
It might be easier to take these as strings and then use logic in the backend to set them to the correct value

Reply:
Fixed